Core/src/main/java/com/plotsquared/core/command/CommandCategory.java

```diff
@@ -95,4 +96,16 @@ public String toString() {
     public String getComponent(@NonNull LocaleHolder localeHolder) {
         return this.caption.getComponent(localeHolder);
     }
+
+    /**
+     * Checks if a player has access to this command category
+     *
+     * @param player The player to check against
+     * @return {@code true} if at least one command of this category can be executed by the player, {@code false} otherwise
+     * @since TODO
+     */
+    public boolean hasPermission(PlotPlayer<?> player) {
```

Comment:
Unless I'm wrong, there's currently no API to register custom categories, therefore it would suffice if this is package private (for now), no?
Maybe it's worth to name this method differently, to prevent possible ambiguity with the `#hasPermission` in Permissions 🤔